.pre-commit-config.yaml

```diff
@@ -18,3 +18,24 @@
   hooks:
   - id: pydocstyle
     exclude: blog/migrations
+
+- repo: git://github.com/pre-commit/mirrors-pylint.git
+  rev: v1.9.1
+  hooks:
+  - id: pylint
+    args:
+    - --msg-template={path}:L {line:3d}:({symbol}) {msg} (C {column:2d}), ::{msg_id}
+    - --output-format=colorized
+    - |-
+        --init-hook=
+        import os, platform, sys;
+        sys.path.insert(
+        0,
+        "".join([os.getenv("HOME"), "/virtualenv/python", platform.python_version(),
+        "/lib/python", ".".join(platform.python_version_tuple()[:2]), "/site-packages"])
+        if os.getenv("TRAVIS") else
+        "~/.virtualenv/django_girls/lib/python3.6/site-packages"
```

Comment:
There must be a way to automatically detect this

Reply:
```
In [1]: import shutil
In [2]: shutil.which("python").split("bin/python")[0] + "lib/python3.6/site-packages"
Out[2]: '/home/sergey/.virtualenv/django_girls/lib/python3.6/site-packages'
```

But pre-commit show me errors with this settings, I don't understand still why is not right.

Reply:
If I write just `"/home/sergey/.virtualenv/django_girls/lib/python3.6/site-packages"` all work.

Reply:
Under pre-commit you'll get pre-commit's virtualenv for pylint, not yours because I'd be injected at the beginning of the `$PATH`. Also, I'd still assume it as hardcoded, since it takes into account only your specific env.

Reply:
Let's leave it as is for now and deal with this later. I'm starting to think that it's better to use pre-commit's features to make it right.